### DISCUSSION TITLE
Create variant using mutation

### DISCUSSION THREAD

**Discussion by Abhijith6252** [pimcore/pimcore#9463] [Q&A / Support]
How to add a variant to an object using mutation?

**Comment by Jimerson21** [comment]
Hello Abhijith6252,

This way you can create variants, I hope it will help you. 

    mutation {
        createProducto(
            parentId: //Product parent ID
            type: "variant"
            key: //SKU product child for example
            input: {
                ...
            }
        )
        {
            success 
            message
            output{
                id:id
            }
        }
    }

**Reply by Abhijith6252** [reply]
> Hello Abhijith6252,
> 
> This way you can create variants, I hope it will help you.
> 
> ```
> mutation {
>     createProducto(
>         parentId: //Product parent ID
>         type: "variant"
>         key: //SKU product child for example
>         input: {
>             ...
>         }
>     )
>     {
>         success 
>         message
>         output{
>             id:id
>         }
>     }
> }
> ```

I am not able to add the type field in the mutation

**Reply by Jimerson21** [reply]
The type field should be available, because it is an attribute of the pimcore system.

![image](https://user-images.githubusercontent.com/58362749/122395990-8bcb9480-cf45-11eb-8742-901f0ac2a39c.png)
